lib/host/aws.js

```diff
@@ -20,11 +20,13 @@ AWS Metadata service endpoint.
 @const
 @see http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AESDG-chapter-instancedata.html
 */
-var BASE_URL = 'http://169.254.169.254/2012-01-12';
+var BASE_URL = 'http://169.254.169.254/latest';
```

Comment:
Yea, this was deliberate.  When I curled for version, the most recent one was 11-05-2014 and this seems to be added after that.